ui_framework/src/components/avatar/avatar.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { VISUALIZATION_COLORS } from '../../services/colors/visualization_colors';
+
+const sizeToClassNameMap = {
+  'none': null,
+  's': 'kuiAvatar--s',
+  'm': 'kuiAvatar--m',
+  'l': 'kuiAvatar--l',
+  'xl': 'kuiAvatar--xl',
+};
+
+export const SIZES = Object.keys(sizeToClassNameMap);
+
+export const KuiAvatar = ({
+  image,
```

Comment:
Can we change `image` to `imageUrl`? This way we can add an `image` prop easily later if we want to allow engineers to pass in an an `img` element.